src/accounts.py

```diff
@@ -0,0 +1,30 @@
+user={"Rhy":"tah"}#key is user, valu is pswd
+accounts = [] #list of users
+
+
+
+def add_account(name,password):
+    print("USER SIGNUP")
+
+    name= str(input("Please enter your name: "))
+    password= str(input("Please enter secret word: "))
+    user=dict()
+
+    if name and password not in user:
+        user['name']=name
+        user['password']=password
+        accounts.append(user)
+        print("Account created")
+        print(accounts)
+    
+
+  
+
+def login(name,password):
+    print("USER LOGIN")
+    name= str(input("Username: "))
```

Comment:
You don't need to cast your input to `str` because the input function by default converts user input to a string.